pyproject.toml

```diff
@@ -1,3 +1,43 @@
+[tool.poetry]
+name = "moabb"
+version = "0.2.1"
```

Comment:
I think you could make it a 0.3.0 revision

Reply:
done